code/modules/reagents/newchem/drugs.dm

```diff
@@ -457,4 +458,173 @@ datum/reagent/fliptonium/overdose_process(var/mob/living/M as mob)
 					M.drop_r_hand()
 					M.adjustToxLoss(1)
 	..()
-	return
\ No newline at end of file
+	return
+
+//////////////////////////////
+//		Synth-Drugs			//
+//////////////////////////////
+
+//Ultra-Lube: Meth
+/datum/reagent/lube/ultra
```

Comment:
Dude, just make these subtypes of the chems they are copypastas of and change the flags on the child you made